exchanges/bybit/bybit_test.go

```diff
@@ -3226,7 +3226,7 @@ func TestWsTicker(t *testing.T) {
 				assert.Equal(t, 6780.866843, v.Volume, "Volume should be correct")
 				assert.Equal(t, "BTC_USDT", v.Pair.String(), "Pair should be correct")
 				assert.Equal(t, asset.Spot, v.AssetType, "AssetType should be correct")
-				assert.Equal(t, int64(233366401000), v.LastUpdated.UnixMilli(), "LastUpdated should be correct")
+				assert.Equal(t, int64(2333664010000), v.LastUpdated.UnixMilli(), "LastUpdated should be correct")
```

Comment:
Please relook at this test. The dates are far in the future. I don't mind if you're just updating the time with a more recent one, but it makes it look like an error

I have verified that the WS data is otherwise coming back properly